litellm/model_prices_and_context_window_backup.json

```diff
@@ -1800,6 +1800,24 @@
         "supports_vision": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "medlm-medium": {
+        "max_tokens": 32768,
```

Comment:
hey @skucherlapati `max_tokens` is the same as `max_output_tokens`. it's an old field based on the openai param of the same name. 

The correct field for input tokens is `max_input_tokens`.

Reply:
i would update this to have `max_tokens`, `max_input_tokens` and `max_output_tokens` for compatibility.

Reply:
Update json to have all 3 keys with `max_tokens == max_output_tokens`.